### PR TITLE
SWTCH 1119 add hints to onchain offchain checkboxes

### PR DESCRIPTION
**Pull request by amaugosomto**

### CODE REVIEW

File: src/app/routes/registration/enrolment-form/enrolment-form.component.html [hunk 1]
```diff
@@ -1,24 +1,27 @@
...
-                      [ngStyle]="txtboxColor">
-                  <span class="mb-3 d-inline-flex"
-                        [ngStyle]="txtColor"
-                        matTooltip="Short description of what is On-chain.">
-                      On-chain
-                  </span>
+            data-qa-id="on-chain"
+            formControlName="onChain"
+            [ngStyle]="txtboxColor"
+        >
```
Comment (DawidGil-jlabs) [inline src/app/routes/registration/enrolment-form/enrolment-form.component.html:7]: Why are you putting closing tag in next line?

File: src/app/routes/registration/enrolment-form/enrolment-form.component.html [hunk 2]
```diff
@@ -1,24 +1,27 @@
...
-                          matTooltip="Short description of what is Off-chain.">
-                        Off-chain
-                    </span>
+            data-qa-id="off-chain"
+            formControlName="offChain"
+            [ngStyle]="txtboxColor">
+        <span class="mb-3 d-inline-flex"
+                [ngStyle]="txtColor"
+                matTooltip="A role recorded in a smart contract that can be read by other smart contracts"
+        s>
```
Comment (DawidGil-jlabs) [inline src/app/routes/registration/enrolment-form/enrolment-form.component.html:22]: `s` before closing tag.

File: src/app/routes/registration/enrolment-form/enrolment-form.component.html (hunk 1 above)
Comment (amaugosomto) [reply src/app/routes/registration/enrolment-form/enrolment-form.component.html:7]: When I use react. Eslint requires the brackets to be on the same line. So I kinda formed the habit.

Comment (amaugosomto) [reply src/app/routes/registration/enrolment-form/enrolment-form.component.html:7]: But for that, I made those changes to force a rebuild.

Comment (VOSpdUkraine) [reply src/app/routes/registration/enrolment-form/enrolment-form.component.html:7]: please use a single standard for this project

File: src/app/routes/registration/enrolment-form/enrolment-form.component.html [hunk 3]
```diff
@@ -1,24 +1,25 @@
...
-                        matTooltip="Short description of what is On-chain.">
-                      On-chain
-                  </span>
+            data-qa-id="on-chain"
+            formControlName="onChain"
+            [ngStyle]="txtboxColor">
+            <span class="mb-3 d-inline-flex"
+                [ngStyle]="txtColor"
+                matTooltip="A JWT verifiable credential that can be presented"
+            >
```
Comment (DawidGil-jlabs) [inline src/app/routes/registration/enrolment-form/enrolment-form.component.html:10]: One more :D

**Review by DawidGil-jlabs — APPROVED** [review]
Lgtm

**Pull request merged**